modules/nextflow/src/main/groovy/nextflow/trace/WebLogObserver.groovy

```diff
@@ -110,18 +118,27 @@ class WebLogObserver implements TraceObserver{
      */
     @Override
     void onFlowStart(Session session) {
-        // This is either set by the user or via Nextflow name generator
+        this.session = session
+        setUpMetadataFromSession()
+
+        asyncHttpMessage("started", createFlowPayloadFromSession(session))
+    }
+
+    private void setUpMetadataFromSession() {
+        storeWorklowMetadataFromSession()
+    }
+
+    private void storeWorklowMetadataFromSession(){
         runName = session.getRunName()
```

Comment:
There's a typo in `storeWorklowMetadataFromSession`. Above all I find redundant two functions for two lines of code 

```
    private void setUpMetadataFromSession() {
        storeWorklowMetadataFromSession()
    }

    private void storeWorklowMetadataFromSession(){
        runName = session.getRunName()
        runId = session.getUniqueId()
    }
```

It would not be better and more readable, just: 

```
    void onFlowStart(Session session) {
        this.session = session
        runName = session.getRunName()
        runId = session.getUniqueId()

        asyncHttpMessage("started", createFlowPayloadFromSession(session))
    }

Reply:
thank you, some leftover after a refactoring. I reduce it